locales/templates/app.pot

```diff
@@ -1954,22 +1953,35 @@ msgid ""
 msgstr ""
 
 msgctxt "msrp_challenge_details_title_2"
-msgid "2. Use your Firefox account to redeem your reward"
+msgid "Use your Firefox account to redeem your reward"
 msgstr ""
 
+#. The %1$s here is a placeholder for "Enter your email address to get
+#. started."
+#, c-format
 msgctxt "msrp_challenge_details_body_2"
 msgid ""
-"Enter your email address to get started. Make sure you sign in to the "
-"account you want to use before you complete your redemption."
+"%1$s Make sure you sign in to the account you want to use before you "
+"complete your redemption."
+msgstr ""
+
+msgctxt "msrp_challenge_details_sign_in_to_start"
+msgid "Enter your email address to get started."
 msgstr ""
 
 #. "I'm In!" is a button where users can press to join the reward program
 msgctxt "msrp_challenge_button_1"
 msgid "I’m In!"
 msgstr ""
 
+#. The %1$s here is a placeholder for "Terms of Use"
+#, c-format
 msgctxt "msrp_challenge_tou"
-msgid "By clicking “I’m in” you are agreeing to our Terms of Use"
+msgid "By clicking “I’m In!” you are agreeing to our %1$s"
```

Comment:
Are we sure this should be a placeholder? I won't block on this, but if this isn't supposed to be a placeholder, let's get in a fix for that quickly :) (it's not a placeholder in the next string to give an example)